core/src/main/java/hudson/cli/CreateViewCommand.java

```diff
@@ -38,6 +41,7 @@
 public class CreateViewCommand extends CLICommand {
 
     @Argument(usage = "Name of the view to use instead of the one in XML")
```

Comment:
Could you make it explicit here that any `/` in the name will be used to put the view into a folder?